src/App.js

```diff
@@ -1,13 +1,37 @@
 import React, {useState} from "react"
 import './App.css';
+import Input from "./Input";
 
 function App() {
   const [formData, setFormData] = useState({
-    name: ""
+    name: "",
+    message: ""
   })
 
+  const handleChange = e => {
+    setFormData({
```

Comment:
Good logic!
I would pass the key (name, message, etc) as a parameter.
`const handleChange = (e, key) => {}` instead of e.target.name, a lot easier to read.
Also, put the updated state in its own variable:
```
const newData = {
      ...formData,
      [key]: e.target.value
    };
    setFormData(newData);
```